.github/workflows/build-and-publish-docker-image.yml

```diff
@@ -0,0 +1,34 @@
+name: Docker Image CI
+
+on:
+  workflow_dispatch:
```

Comment:
This means that the workflow can also be manually triggered on a user selectable branch btw: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Reply:
yes, I had hoped that would be the case to help testing. But I can't see the button yet...once it is working, we can remove this.